lms/static/sass/discussion/views/_thread.scss

```diff
@@ -207,3 +203,10 @@ body.view-in-course .discussion-article {
     }
   }
 }
+
+// Fix for overflowing images
+.thread-wrapper {
```

Comment:
There's already a top-level ``.thread-wrapper`` rule in here, so can you merge the two.